source/connection-monitoring-and-pooling/connection-monitoring-and-pooling.rst

```diff
@@ -502,11 +547,14 @@ semaphore, a condition variable may also be needed to to meet this
 requirement. Waiting on the condition variable SHOULD also be limited by the
 WaitQueueTimeout, if the driver supports one and it was specified by the user.
 
-If the pool is closed, any attempt to check out a `Connection <#connection>`_ MUST throw an Error, and any items in the waitQueue MUST be removed from the waitQueue and throw an Error.
+If the pool is "closed" or "paused", any attempt to check out a `Connection
+<#connection>`_ MUST throw an Error, and any items in the waitQueue MUST be
+removed from the waitQueue and throw an Error. The error thrown as a result of
+the pool being "paused" MUST be retryable.
```

Comment:
I wanted to add a unified format test to the retryable-reads spec for this, but they don't have any unified format tests yet. Should I just write a prose test for it now that can be converted once that spec is moved to unified format, just write the unified format one now, or don't write any test and wait? All seem reasonable to me.